src/Engine/ProtoCore/Utils/MathUtils.cs

```diff
@@ -4,7 +4,7 @@ namespace ProtoCore.Utils
 {
     public static class MathUtils
     {
-        public static double Tolerance = 1e-5;
```

Comment:
is this an API break?
I guess the value is still accessible inside the static class?